packages/rolldown/tests/fixtures/builtin-plugin/ecma-transform/exclude/_config.ts

```diff
@@ -8,7 +8,7 @@ export default defineTest({
     input: './main.ts',
     plugins: [
       transformPlugin({
-        exclude: ['node_modules/**'],
+        exclude: ['**/node_modules/**'],
```

Comment:
rollup createFilter return false for `node_modules/**` either
```js
import {createFilter} from '@rollup/pluginutils'


const test = createFilter('node_modules/**')
console.log(`test('/virtual/foo/node_modules/bar.js'): `, test('/foo/bar/node_modules/bar.js'))

```

test('/virtual/foo/node_modules/bar.js'):  false